src/editor.css

```diff
@@ -173,3 +173,7 @@
 .file-list-empty > div {
   margin: 10px 0;
 }
+
+.template {
+  width: 20%;
```

Comment:
Hmm, maybe these previews are too small depending on screen size? Let's just force them to `100px` width, try out how that looks.

Reply:
Actually, disregard that. Since we just have 5, setting it to 20% is useful and clever :) Let's leave as-is for now, no `overflow`.